### ISSUE TITLE
 Intel(R) N150 Error with I915MT65-24.1.19-5

### ISSUE THREAD

**Issue by CWWK-TechnicalSupport** [moetayuko/intel-gpu-i915-backports#34]
Linux CW-TWLN-4L 6.8.12-5-pve #1 SMP PREEMPT_DYNAMIC PMX 6.8.12-5 (2024-12-03T10:26Z) x86_64 GNU/Linux

Building initial module for 6.8.12-5-pve
Error! Bad return status for module build on kernel: 6.8.12-5-pve (x86_64)
Consult /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/build/make.log for more information.
dpkg: error processing package intel-i915-dkms (--install):
 installed intel-i915-dkms package post-installation script subprocess returned error exit status 10
Errors were encountered while processing:
 intel-i915-dkms

**Comment by CWWK-TechnicalSupport** [comment]
DKMS make.log for intel-i915-dkms-1.24.1.19.240119.1.nodrm for ke>
Tue Dec 10 11:33:47 AM CST 2024
make[1]: Nothing to be done for 'updateconfig'.
cc -Wall -Wmissing-prototypes -Wstrict-prototypes -O2 -fomit-fram>
flex -ozconf.lex.c -L zconf.l
bison -ozconf.tab.c -t -l zconf.y
cc -Wall -Wmissing-prototypes -Wstrict-prototypes -O2 -fomit-fram>
cc   conf.o zconf.tab.o   -o conf
#
# configuration written to .config
#
make[2]: Nothing to be done for 'updateconfig'.
make[4]: Nothing to be done for 'updateconfig'.
make[5]: 'conf' is up to date.
#
# configuration written to .config

#
Building backport-include/backport/autoconf.h ... done.
Building backport-include/backport/backport_path.h ... done.
cat: versions: No such file or directory
cat: versions: No such file or directory
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
/var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/build/comp>
   10 | u8 dp_link_status(const u8 link_status[DP_LINK_STATUS_SIZ>
      |    ^~~~~~~~~~~~~~
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  LD [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
/var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/build/driv>
/var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/build/driv>
  437 |                 if (pm_runtime_get_if_active(rpm->kdev, i>
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~
In file included from /var/lib/dkms/intel-i915-dkms/1.24.1.19.240>
                 from /var/lib/dkms/intel-i915-dkms/1.24.1.19.240>
./include/linux/pm_runtime.h:75:12: note: declared here
   75 | extern int pm_runtime_get_if_active(struct device *dev);
      |            ^~~~~~~~~~~~~~~~~~~~~~~~
  CC [M]  /var/lib/dkms/intel-i915-dkms/1.24.1.19.240119.1.nodrm/>
make[6]: *** [scripts/Makefile.build:243: /var/lib/dkms/intel-i91>
make[6]: *** Waiting for unfinished jobs....
make[5]: *** [scripts/Makefile.build:481: /var/lib/dkms/intel-i91>
make[4]: *** [Makefile:1925: /var/lib/dkms/intel-i915-dkms/1.24.1>
make[3]: *** [Makefile.build:6: modules] Error 2
make[2]: *** [Makefile.real:95: modules] Error 2
make[1]: *** [Makefile:69: modules] Error 2
make: *** [Makefile:52: default] Error 2

**Comment by PeterSuh-Q3** [comment]
Currently, the only solution is to use Ubuntu 24.04.02 or 24.04.03 with kernel 6.9 or later within Proxmox LXC.

Only in this case will /dev/dri be enabled on the latest n150 (Twin Lake) CPU.

Supporting this requirement on an Epyc7002 (sa6400) baremetal with kernel 5.10 be challenging.

The following script exists for easy transcoding within LXC.

[Plex]
```
bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/plex.sh)"
```

[Jellyfin]
```
bash -c "$(curl -fsSL https://raw.githubusercontent.com/PeterSuh-Q3/ProxmoxVE/main/ct/jellyfin.sh)"
```
